include/zephyr/arch/x86/ia32/arch.h

```diff
@@ -51,6 +51,12 @@
 #define GS_TLS_SEG	(0x18 | 0x03)
 #endif
 
+#if CONFIG_THREAD_LOCAL_STORAGE
+#define PF_TSS		(GS_TLS_SEG + 8)
```

Comment:
(GS_TLS_SEG & 0xF8), privilege setting of GS_TLS_SEG should be removed.

Reply:
good catch, fixed with new commit, thanks.